lib/cereal/builders/entity.ex

```diff
@@ -27,6 +27,9 @@ defmodule Cereal.Builders.Entity do
 
   # if an assigns/2 function was specified in the serializer - we'll add its
   # return value to the conn's assigns
+  defp do_assigns(%{conn: conn} = context) when map_size(conn) == 0 do
```

Comment:
I believe it would be more idiomatic to pattern match that the `conn` attribute in the context map is indeed a `Plug.Conn` struct.

Reply:
ah yeah definitely - updated!